tests/test_change_graphs.py

```diff
@@ -413,6 +413,19 @@ def skew(self, axis=0, skipna=True, level=None):
 
 
 if __name__ == '__main__':
+    PYTHON_BIN_ENV_VAR = "GUMTREE_PYTHON_BIN"
+    PYPARSER_ENV_VAR = "GUMTREE_PYPARSER_PATH"
+    PYTHON_BIN_VALUE = "python3"
+
+    project_directory = os.path.dirname(os.getcwd())
+    pyparser_path = project_directory + '/external/pyparser.py'
+
+    if os.getenv(PYTHON_BIN_ENV_VAR) is None:
+        os.environ[PYTHON_BIN_ENV_VAR] = PYTHON_BIN_VALUE;
+
+    if os.getenv(PYPARSER_ENV_VAR) is None:
+        os.environ[PYPARSER_ENV_VAR] = pyparser_path;
+
```

Comment:
Maybe move it to a separate method in the `main.py` to avoid copypasting?

Reply:
done

Reply:
Nice! But I actually meant to move all the logic for setting env variables to another function (maybe even in some module like `deployment.py`). So we won't need to copy-paste all these lines each time (in `main.py`, in tests, ...)